plugin/inlay_hint.py

```diff
@@ -140,13 +142,23 @@ def format_inlay_hint_label(inlay_hint: InlayHint, session: Session, phantom_uui
                 }
             })
             result += f'<a href="{inlay_hint_click_command}">'
+        truncated = len(label) >= truncate_limit and truncate_limit > 0
+        truncated_label = label[:truncate_limit] + '…' if truncated else label
+
```

Comment:
Shouldn't it be `>` instead of `>=`? If the label length is exactly `truncate_limit`, you don't need to add the dots. Also I think it should be `label[:truncate_limit - 1]`, so that the length with added dots doesn't exceed the `"inlay_hints_max_length"` value.

The empty line should be removed (also in line 127).

Reply:
> Shouldn't it be `>` instead of `>=`? If the label length is exactly `truncate_limit`, you don't need to add the dots. Also I think it should be `label[:truncate_limit - 1]`, so that the length with added dots doesn't exceed the `"inlay_hints_max_length"` value.
> 
> The empty line should be removed (also in line 127).

Please see: https://github.com/sublimelsp/LSP/pull/2514#discussion_r1779999461

Reply:
Empty lines are removed in: [b83a147](https://github.com/sublimelsp/LSP/pull/2514/commits/b83a1474059562bc0a72c7c68947aa3777b8822a)

I am unsure if we should change `>` into `>=`, because it was already reviewed, and requested to change into `>=`.

Reply:
Done in: [9be92d9](https://github.com/sublimelsp/LSP/pull/2514/commits/9be92d9085791260140dc74a96ec3670a313fbc0)